portal/templates/dashboard/_macros/dashboard-deals-table.njk

```diff
@@ -27,7 +27,7 @@
         <a
           href="{{ url }}{{ deal._id }}"
           class="govuk-link"
-          data-cy="deal__link--{{ deal._id }}"
+          data-cy="deal__link--{{ deal._id }} deal__link--index--{{ deal._id }}"
```

Comment:
This has been changed here but not changed in the relevant E2E pages/partials section.  Is there a reason why?

Reply:
Sorry, this was a mistake after stashing and switching branches, have removed.